mobile_app/README.md

```diff
@@ -11,15 +30,16 @@ This will:
   named CONFIGURATION.
 * Create a new commit with that environment and push to origin.
 
-This will in turn trigger a CI job which makes a new build with that commit,
-providing access to that environment. The goal is to let a job be triggered in
-various ways, particularly jenkins, but have the actual build step be run on a
-separate build machine.
+The build repo should be configured so that this push will in turn trigger a CI
+job which makes a new build with that commit, providing access to that
+environment. The goal is to capture the environmment where the job is triggered
+(like Jenkins) and save it so the actual build step can be run on a separate
+build machine with access to that environment.
 
 *Usage*
 
 * Create your virtual environment
-* pip install -r mobile_app/requirements.txt
+* ``pip install -r mobile_app/requirements.txt``
```

Comment:
nit: some of these steps are repeated across the different scripts. Could the 'set up' part of 'Usage' be pulled into a different section to lighten up the page?